bot.py

```diff
@@ -28,8 +30,14 @@ async def on_command_error(ctx, error):
 		return
 	await ctx.send("An error occured!\n```{}```".format(error))
 '''
-		
-for cog in os.listdir(".\\commands"):#path
+
+curdir = os.getcwd()
+curdir.split('\\')
+if "Synergile" not in curdir:
+	os.chdir("Synergile")
+else:
+	pass
```

Comment:
```suggestion
```
Hardcoding things like this tends to have negative effects in a distributed context